index.html

```diff
@@ -2106,6 +2106,11 @@ <h3>Processing capabilities</h3>
        used. The intent is that if this is not possible a new session
        will not be established.
 
+     <dt>"<code>setWindowRect</code>"
+     <dd><p>return <a>success</a> with data <a><code>null</code></a>.
+
+      <p class=note>Matching is not supported for this capability.
```

Comment:
The referenced issue actually requests matching, and I've given an example with some more details. So I think that not matching the capability is not what we probably want.

I would suggest to get it discussed on the issue which direction we want to go before continuing here?